### DISCUSSION TITLE
Light theme issue

### DISCUSSION THREAD

**Discussion by ItsRobbAllen** [ravendb/ravendb#14357] [General]
In what seems to be offensive to most devs, I don't use Dark Mode on 100% of my apps. Mostly because everything being a dull gray, it gets difficult to find which window I need, but also because I am old & my eyes aren't all that great.

So, for Raven, I use the light theme. There's a problem though when I go to test ETLs. The contrast between the background & the variable names is too low & it's difficult to see.

<img width="353" alt="image" src="https://user-images.githubusercontent.com/32247592/172413959-452e1e3e-d5e9-434f-afcf-7d8b42433239.png">

Could the background be turned white or the variable names darkened considerably?

Thanks,
R

**Comment by garayx** [comment]
Hi

Thanks for reporting this, we will fix it, please follow this ticket for progress: https://issues.hibernatingrhinos.com/issue/RavenDB-18748

**Comment by ItsRobbAllen** [comment]
Thanks for looking out for us old guys :D
